instrumentation/undertow-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/undertow/UndertowHttpAttributesGetter.java

```diff
@@ -55,7 +55,8 @@ public String getUrlPath(HttpServerExchange exchange) {
   @Nullable
   @Override
   public String getUrlQuery(HttpServerExchange exchange) {
-    return exchange.getQueryString();
+    String queryString = exchange.getQueryString();
+    return !"".equals(queryString) ? queryString : null;
```

Comment:
Nit: can you add a comment here? e.g. `undertow returns empty string even when query was not set at all`